public/inbox_sent_view.php

```diff
@@ -0,0 +1,86 @@
+<?php
+require_once '../common.php';
+require_once '../modules/models/user.php';
+require_once '../modules/models/email.php';
+require_once '../modules/models/inbox.php';
+require_once '../modules/models/sent.php';
+?>
+
+<?php
+// Get the logged in user
+if (isset($_SESSION['current_user'])) {
+    $current_user = $_SESSION['current_user'];
+} else {
+    echo 'Error fetching logged in user.';
+    header('Location: login.php');
+}
+// Fetch either inbox or sent, depending on what user specified
+if ($_SESSION["email_view"] == "inbox") {
+    $box_entries = Inbox::where(array('email_address' => $current_user->email));
+} elseif ($_SESSION["email_view"] == "sent") {
+    $box_entries = Sent::where(array('email_address' => $current_user->email));
+}
+// Sort the messages by sent date/time
+$by_created_at = array();
+foreach ($box_entries as $be) {
+    $by_created_at[$be->id] = $be->created_at;
+}
```

Comment:
🤔 hmm i think you could extend the query builder to do this, i.e. make it possible to do ORDER BY (look at how I handled LIMIT)